src/pages/workspaces/workspace/WorkspaceContainer.ts

```diff
@@ -49,6 +49,7 @@ const TitleBarWarning = (props: PropsWithChildren): ReactNode => {
     ),
     style: { backgroundColor: colors.accent(0.35), borderBottom: `1px solid ${colors.accent()}` },
     onDismiss: () => {},
+    hideCloseButton: true,
```

Comment:
The close button doesn't do anything on this component (note the empty function body of `onDismiss`), so this is just a minor change to make it behave more consistently.